package.json

```diff
@@ -2,7 +2,7 @@
     "name": "qs",
     "description": "A querystring parser that supports nesting and arrays, with a depth limit",
     "homepage": "https://github.com/ljharb/qs",
-    "version": "6.11.2",
+    "version": "6.12.2",
```

Comment:
versions should never be updated in PRs, and when incrementing the minor, the patch must go to 0 anyways.

Reply:
this still needs to be reverted